### ISSUE TITLE
QA surfaced KiteRequestError

### ISSUE THREAD

**Issue by dbratz1177** [kiteco/atom-plugin#482]
@abe33 not sure if this is actually something to fix or not, since it doesn't seem to have any performance impact, but thought it'd be useful to post in any case
<img width="1152" alt="screen shot 2018-08-30 at 1 23 14 pm" src="https://user-images.githubusercontent.com/6146901/44877189-083b3e80-ac58-11e8-900c-4fb9d33d1a3c.png">

**Comment by abe33** [comment]
Hmm, yeah these errors are now handled and reported through rollbar (hence why it appears in the stack) so it's why it doesn't break the plugin, it's just more verbose because rollbar logs every errors in the console. 

Now there's still something to fix if we can't parse a JSON response (we're trying to parse something because we expect a response here). So far we're using that `requestJSON` function for these calls: 
- /clientapi/user
- /clientapi/languages
- all data apis calls (hover, completions, signatures, symbol/value reports, etc.)
- /api/account/user
- /clientapi/status

Maybe some information about the context in which this occured would help me track this down in our APIs (the sad state of JS makes it hard to follow errors when promise/async stuff is involved)

**Comment by dbratz1177** [comment]
> (the sad state of JS makes it hard to follow errors when promise/async stuff is involved)
🤣 tell me about it
I'll keep an eye out for similar reports coming back from QA to try and get some more context next time it emerges.
